### DISCUSSION TITLE
Finding this driver MADE MY DAY, thanks so much!

### DISCUSSION THREAD

**Discussion by moodboom** [FreeSpacenav/spacenavd#34] [Questions and support]
It is working perfectly with ubuntu 20.04, systemd service, blender.  I followed this [guy's instructions](https://mattjones.tech/2019/03/26/using-3dconnexion-spacemouse-with-linux/#respond), so easy.

**Comment by jtsiomb** [comment]
Excellent, glad to hear it.

**Comment by r-lapins** [comment]
Fedora 41 Plasma KDE Desktop - WORKS with Blender and FreeCAD!
THANK YOU!
